bom/application/pom.xml

```diff
@@ -39,11 +39,12 @@
         <microprofile-rest-client.version>2.0</microprofile-rest-client.version>
         <microprofile-jwt.version>1.2</microprofile-jwt.version>
         <microprofile-lra.version>1.0</microprofile-lra.version>
+        <microprofile-openapi.version>2.0.1</microprofile-openapi.version>
         <smallrye-common.version>1.13.2</smallrye-common.version>
         <smallrye-config.version>2.13.0</smallrye-config.version>
         <smallrye-health.version>3.3.1</smallrye-health.version>
         <smallrye-metrics.version>3.0.5</smallrye-metrics.version>
-        <smallrye-open-api.version>2.3.1</smallrye-open-api.version>
+        <smallrye-open-api.version>3.1.1</smallrye-open-api.version>
```

Comment:
So this version will work in `main` with the `javax` packages?

Reply:
That's correct. The scanning is all Jandex-based and both top-level packages are scanned.

Reply:
If `main` includes the Jakarta migration (or even MP 6), I'll need to update the version for `microprofile-openapi.version`